src/System.Diagnostics.Process/tests/ProcessWaitingTests.cs

```diff
@@ -141,7 +143,7 @@ public void SingleProcess_CopiesShareExitInformation()
         }
 
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/corefx/issues/22269", TargetFrameworkMonikers.UapNotUapAot)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapNotUapAot, "Getting handle of child process running on UAP is not possible")]
```

Comment:
this is an app/appcontainer issue, not a coreclr issue, so is `TargetFrameworkMonikers.Uap` more appropriate? Then when we run ILC tests in appcontainer later, it will not fail.